packages/react-components/react-progress/package.json

```
@@ -36,6 +36,7 @@
     "@fluentui/react-shared-contexts": "^9.3.3",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

_nit:_ Can you move this above `@fluentui/react-shared-contexts` in line `36` to keep alphabetical order?

```suggestion
    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
    "@fluentui/react-shared-contexts": "^9.3.3",
    "@fluentui/react-theme": "^9.1.7",
    "@fluentui/react-utilities": "^9.8.0",
```